src/lib/components/lines-and-dots/svg/history-graph-row.svelte

```diff
@@ -59,7 +60,9 @@
     point={[visualWidth + 50, y]}
     category={event.category}
     active={noActives || isActiveEvent}
-    icon={CategoryIcon[event.category]}
+    icon={isLocalActivityMarkerEvent(event)
+      ? 'feather'
+      : CategoryIcon[event.category]}
```

Comment:
Would it make sense to add
```js
'local-activity': 'feather',
```
to `CategoryIcon` and use `CategoryIcon['local-activity']` here instead?

Reply:
Good call